index.html

```diff
@@ -0,0 +1,122 @@
+            <h1>Welcome to our portfolio!</h1>
+            <p>
+                &copy; <a href="https://github.com/jorgeabrahan" class="text-light">Jorge</a> &
+                <a href="https://github.com/danielamoreno699" class="text-light" id="profile-link">Daniela</a>
+            </p>
+        </section>
+
+        <section id="projects" class="container mb-5">
+            <h3 class="fs-1 fw-bold text-center mb-4">Some of our projects</h3>
+            <div class="card-group gap-3" id="project-tile">
+                <a
+                    class="card"
+                    style="cursor: pointer; text-decoration: none; color: black"
+                    href="https://calculadora-38eda.web.app/"
+                >
+                    <img src="img/calculator.jpg" class="card-img-top" alt="..." style="height: 100%" />
+                    <div class="card-body">
+                        <h5 class="card-title">JS calculator</h5>
+                        <p class="card-text">
+                            Lorem ipsum dolor sit, amet consectetur adipisicing elit. Commodi, molestiae.
+                        </p>
+                    </div>
+                </a>
+                <a
+                    class="card"
+                    style="cursor: pointer; text-decoration: none; color: black"
+                    href="https://budgtcalc.web.app/"
+                >
+                    <img src="img/finance.jpg" class="card-img-top" alt="..." style="height: 100%" />
+                    <div class="card-body">
+                        <h5 class="card-title">JS budget calculator</h5>
+                        <p class="card-text">
+                            Lorem ipsum dolor sit, amet consectetur adipisicing elit. Commodi, molestiae.
+                        </p>
+                    </div>
+                </a>
+                <a
+                    class="card"
+                    style="cursor: pointer; text-decoration: none; color: black"
+                    href="https://firestore-grafico-dcd5e.web.app/inicio"
+                >
+                    <img src="img/games.jpg" class="card-img-top" alt="..." style="height: 100%" />
+                    <div class="card-body">
+                        <h5 class="card-title">Games survey</h5>
+                        <p class="card-text">
+                            Lorem ipsum dolor sit, amet consectetur adipisicing elit. Commodi, molestiae.
+                        </p>
+                    </div>
+                </a>
+            </div>
+        </section>
+
+        <section class="container">
+            <div class="row">
+                <a href="#" class="col fa fa-facebook"></a>
+                <a href="#" class="col fa fa-twitter"></a>
+                <a href="#" class="col fa fa-instagram"></a>
+                <a href="#" class="col fa fa-linkedin"></a>
+            </div>
```

Comment:
I suggest putting all social icons inside a UL/LI listing, this is a good practice to ensure that the user always sees the content in a listing format. Also, make sure links go somewhere if they are external links I kindly suggest adding the target attribute to be "_blank". 😃